s3/src/main/java/com/instaclustr/kafka/connect/s3/sink/BeginOffsetTotalRecords.java

```diff
@@ -0,0 +1,29 @@
+package com.instaclustr.kafka.connect.s3.sink;
+
+public class BeginOffsetTotalRecords {
```

Comment:
I am looking in Restore to achieve same logic. Once i get the same logic then i can delete class.